docs/reference/reference-deploy/config/master-config-reference.rst

```diff
@@ -266,10 +266,11 @@ The master supports the following configuration settings:
          Determined master.
 
       -  ``slot_type``: The default slot type assumed when users request resources from Determined
-         in terms of ``slots``. Defaults to ``cuda`` for partitions where GPUs are detected
-         automatically, else ``cpu``. If GPUs cannot be detected automatically, for example when
-         operating with ``gres_supported: false``, then this result may be overridden using
-         ``partition_overrides``.
+         in terms of ``slots``. Available values are ``cuda``, ``rocm`` and ``cpu``, where 1
+         ``cuda`` or ``rocm`` slot is 1 GPU and 1 ``cpu`` slot is 1 node. Defaults per-partition to
```

Comment:
| 1 ``cpu`` slot is 1 node.

This it the default, but you can now change that with slots_per_node.   We should re-word this to cover that case now.

Reply:
How about we amend to:

...and 1 `cpu` slot is 1 node, _unless overridden by `slots_per_node`._

Reply:
Kind of right.    I couldn't come up with precise language on the fly, so I thought I'd leave you to propose :-).  I would approve that, but it seems slightly imprecise because of the units, but it conveys the proper caveat.  What about something like:

`otherwise it requests cpu slots.   The number of cpus allocated per node is 1, unless overridden by slots_per_node`